### DISCUSSION TITLE
197

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#176] [General]
# 潮流周刊第197期 - 喂流浪猫

封面图拍摄于小区散步经常看到的一个流浪猫，长得很像我家汤圆，于是试着带着猫粮下去看能否偶遇，结果遇到了，给他吃了一个饱。

https://weekly.tw93.fun/posts/197-%E5%96%82%E6%B5%81%E6%B5%AA%E7%8C%AB/

**Comment by huanfe1** [comment]
没想到能在这看见自己的文章被推荐，感谢 :smiley:

**Reply by xxholly32** [reply]
respect

**Comment by Stellanliu** [comment]
那个流程图的动态箭头是怎么做的

**Comment by Stefanieissun** [comment]
同问，动态箭头是怎么实现的
